src/web-components/index.js

```diff
@@ -0,0 +1,4 @@
+import { VFScore } from './vf-score';
```

Comment:
I'd probably make these `export`s so that if someone did want to import the classes from this module, they could.